### DISCUSSION TITLE
Volar does not have vetur's built-in snippets

### DISCUSSION THREAD

**Discussion by HenrijsS** [nuxt/nuxt#18029] [General]
### Environment
------------------------------
- Operating System: `Darwin`
- Node Version:     `v14.17.0`
- Nuxt Version:     `3-3.0.0-27234368.47ea41d`
- Package Manager:  `npm`
- Bundler:          `Webpack`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`
------------------------------
### Describe the bug and expected behavior
The docs have 3 downloads in the recommended downloads section and one of them is Volar.
After uninstalling Vetur and installing Volar, all emmet snippets just disappeared. 

### Steps to reproduce
1. Uninstall Vetur
2. Install Volar
3. Use emmet on a fresh Nuxt3 installation

### Additional context
The Volar GitHub repo is pretty inactive in the Issue department, but I also posted the issue there: https://github.com/johnsoncodehk/volar/issues/585

I'm just wondering if anyone else is experiencing this issue.

**Comment by danielroe** [comment]
@HenrijsS Can you replicate this on a Vue3 project without nuxt?

**Comment by HenrijsS** [comment]
@danielroe Same thing when working with Vite.
Are you not experiencing this issue? I'm on Volar@0.28.3
Maybe I need to downgrade if it's a new issue. 

I've also tried disabling all extensions but it still doesn't work.

**Comment by danielroe** [comment]
In that case it's likely a volar question (or perhaps just a feature that vetur enabled which is now missing) - rather than an issue with nuxt 3.

**Reply by HenrijsS** [reply]
I've just installed VScode on an empty laptop and Volar is still not working.
Does it require some special configuration? I'm at a complete loss. 

Please help. ☹️

**Reply by danielroe** [reply]
Seems to be working fine for me.

![CleanShot 2021-10-12 at 21 38 54](https://user-images.githubusercontent.com/28706372/137025841-1740bdb8-23c3-493a-9327-614c73d914b3.gif)

**Reply by HenrijsS** [reply]
![Oct-12-2021 23-44-24](https://user-images.githubusercontent.com/49265222/137026475-b9e91b31-be12-410e-b582-816736d1e903.gif)

The default ones work, but not the Vue specific ones.

**Reply by danielroe** [reply]
I think this is just a volar question.

**Comment by HenrijsS** [comment]
Is like nobody else experiencing this?
I've installed a fresh install of VSCode, Cleared all the config files, Installed only Volar and it still doesn't work.
Don't mind that I also tried the last 8 version of the extension and none of them have Emmet working.

Is this a macOS bug?

**Comment by johnsoncodehk** [comment]
> https://github.com/nuxt/framework/discussions/845#discussioncomment-1467043

Emmet seems working good to you.

Just in case, are you talking about this?

<img width="732" alt="螢幕截圖 2021-10-13 下午1 18 16" src="https://user-images.githubusercontent.com/16279759/137071752-56914de0-50be-47bc-a31a-202e5ba81e06.png">

This is vetur built-in snippets not emmet.

**Reply by HenrijsS** [reply]
But it says on your description that I need to disable Vetur.

**Reply by danielroe** [reply]
Could you explain more about what your question is @HenrijsS?

**Reply by HenrijsS** [reply]
To access these snippets!
It's really tedious to create hundreds of components without these snippets.

[Volar's extension page](https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.volar) shows that it needs to disable Vetur, but then these are Vetur's snippets and not Volar's?
<img width="320" alt="image" src="https://user-images.githubusercontent.com/49265222/137101835-c0e2c69f-67a8-4ae0-95a9-481eb4054161.png">


Maybe I was writing in incorrectly. Emmet is just the word I use for auto-code-completion.

**Reply by danielroe** [reply]
I think it might be a good idea to create a feature request on https://github.com/johnsoncodehk/volar then 🙃

**Comment by gluharry** [comment]
Same here with me
